File: CoreJava/vehical-interface-demo-app/src/com/techlab/vehicals/Car.java
package com.techlab.vehicals;

import com.techlab.imoveable.IMoveable;

public class Car implements IMoveable {
	
	@Override
	public void start() {
		System.out.println("Car Starts....");
	}

	@Override
	public void stop() {
		System.out.println("Car Stops....");
	}

}

File: C#Basics/Interface-App/Vehical-App/Car.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Vehical_App
{
    class Car : IMoveable
    {
		void IMoveable.start()
		{
			Console.WriteLine("Car Starts....");
		}

		
		void IMoveable.stop()
		{
			Console.WriteLine("Car Stops....");
		}

	}
}
